site/using.md

```diff
@@ -13,188 +13,166 @@ the command line. The `--help` for `fluxctl` is described below.
 fluxctl helps you deploy your code.
 
 Workflow:
-  fluxctl list-services                                        # Which services are running?
-  fluxctl list-images --service=default/foo                    # Which images are running/available?
-  fluxctl release --service=default/foo --update-image=bar:v2  # Release new version.
+  fluxctl list-controllers                                           # Which controllers are running?
+  fluxctl list-images --controller=deployment/foo                    # Which images are running/available?
+  fluxctl release --controller=deployment/foo --update-image=bar:v2  # Release new version.
 
 Usage:
   fluxctl [command]
 
 Available Commands:
-
-Having no deployment side effect
-
-  version       Output the version of fluxctl
-  identity      Display SSH public key
-  list-images   Show the deployed and available images for a service.
-  list-services List services currently running on the platform.
-  save          save service definitions to local files in platform-native format
-
-With side effect
-
-  automate      Turn on automatic deployment for a service.
-  deautomate    Turn off automatic deployment for a service.
-  lock          Lock a service, so it cannot be deployed.
-  release       Release a new version of a service.
-  unlock        Unlock a service, so it can be deployed.
+  automate         Turn on automatic deployment for a controller.
```

Comment:
Might be better to group related commands instead of sorting them lexicographically. Specifically lock/unlock and automate/deautomate.

Reply:
This is the order they're printed out by `spf13/cobra`.

Reply:
Fair enough.